modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/logit/LogitLogStorageFactory.java

```diff
@@ -92,6 +92,11 @@ public void close() {
         ExecutorServiceHelper.shutdownAndAwaitTermination(checkpointExecutor);
     }
 
+    @Override
+    public void sync() {
+        // No-op.
```

Comment:
Please throw an exception instead. It's not a no-op.

Reply:
fixed, added a todo